go.mod

```diff
@@ -39,14 +42,14 @@ require (
 	github.com/envoyproxy/protoc-gen-validate v0.2.0-java
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/getsentry/raven-go v0.2.0
-	github.com/go-redis/redis v0.0.0-20190503082931-75795aa4236d
```

Comment:
Can we leave this the way it was?

Reply:
This is caused by `go get github.com/goreleaser/goreleaser@v0.117.1`
I have no idea why, but I think in this case `go` knows better :man_shrugging: